package.json

```diff
@@ -140,7 +142,7 @@
 		"eslint-config-google": "^0.14.0",
 		"eslint-plugin-ava": "^13.0.2",
 		"eslint-plugin-jsdoc": "^37.6.3",
-		"esmock": "^2.0.0",
+		"esmock": "^2.0.7",
```

Comment:
this is unrelated... I just tried to use esmock and wanted to ensure I have the latest version. But it was still not working the way I wanted. See comments in the "integration" tests.